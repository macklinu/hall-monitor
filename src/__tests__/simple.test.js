import hallMonitor from '..'

const rule = context => ({
  Identifier(node) {
    if (node.name === 'foo') {
      context.report({
        node,
        message: 'No variables named foo',
      })
    }
  },
})

hallMonitor('simple', rule, {
  valid: ['', 'let a'],
  invalid: ['let foo = 1'],
})
