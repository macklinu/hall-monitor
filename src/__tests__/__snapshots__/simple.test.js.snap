// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`simple invalid let foo = 1 1`] = `
"let foo = 1
     ^^^ [No variables named foo]"
`;

exports[`simple valid [no code] 1`] = `""`;

exports[`simple valid let a 1`] = `"let a"`;
